2021-02-16 00:39:49,068:INFO:PyCaret Supervised Module
2021-02-16 00:39:49,068:INFO:ML Usecase: regression
2021-02-16 00:39:49,068:INFO:version 2.2.3
2021-02-16 00:39:49,069:INFO:Initializing setup()
2021-02-16 00:39:49,069:INFO:setup(target=count_log, ml_usecase=regression, available_plots={'parameter': 'Hyperparameters', 'residuals': 'Residuals', 'error': 'Prediction Error', 'cooks': 'Cooks Distance', 'rfe': 'Feature Selection', 'learning': 'Learning Curve', 'manifold': 'Manifold Learning', 'vc': 'Validation Curve', 'feature': 'Feature Importance', 'feature_all': 'Feature Importance (All)', 'tree': 'Decision Tree'}, train_size=0.7, test_data=None, preprocess=True, imputation_type=simple, iterative_imputation_iters=5, categorical_features=None, categorical_imputation=constant, categorical_iterative_imputer=lightgbm, ordinal_features=None, high_cardinality_features=None, high_cardinality_method=frequency, numeric_features=None, numeric_imputation=mean, numeric_iterative_imputer=lightgbm, date_features=None, ignore_features=None, normalize=False, normalize_method=zscore, transformation=False, transformation_method=yeo-johnson, handle_unknown_categorical=True, unknown_categorical_method=least_frequent, pca=False, pca_method=linear, pca_components=None, ignore_low_variance=False, combine_rare_levels=False, rare_level_threshold=0.1, bin_numeric_features=None, remove_outliers=False, outliers_threshold=0.05, remove_multicollinearity=False, multicollinearity_threshold=0.9, remove_perfect_collinearity=True, create_clusters=False, cluster_iter=20, polynomial_features=False, polynomial_degree=2, trigonometry_features=False, polynomial_threshold=0.1, group_features=None, group_names=None, feature_selection=False, feature_selection_threshold=0.8, feature_selection_method=classic, feature_interaction=False, feature_ratio=False, interaction_threshold=0.01, fix_imbalance=False, fix_imbalance_method=None, transform_target=False, transform_target_method=box-cox, data_split_shuffle=True, data_split_stratify=False, fold_strategy=kfold, fold=10, fold_shuffle=False, fold_groups=None, n_jobs=-1, use_gpu=False, custom_pipeline=None, html=True, session_id=None, log_experiment=False, experiment_name=None, log_plots=False, log_profile=False, log_data=False, silent=False, verbose=True, profile=False, profile_kwargs=None, display=None)
2021-02-16 00:39:49,072:INFO:Checking environment
2021-02-16 00:39:49,072:INFO:python_version: 3.7.4
2021-02-16 00:39:49,072:INFO:python_build: ('default', 'Aug  9 2019 18:34:13')
2021-02-16 00:39:49,073:INFO:machine: AMD64
2021-02-16 00:39:49,073:INFO:platform: Windows-10-10.0.19041-SP0
2021-02-16 00:39:49,073:INFO:Memory: svmem(total=8504934400, available=2439106560, percent=71.3, used=6065827840, free=2439106560)
2021-02-16 00:39:49,073:INFO:Physical Core: 2
2021-02-16 00:39:49,073:INFO:Logical Core: 4
2021-02-16 00:39:49,073:INFO:Checking libraries
2021-02-16 00:39:49,073:INFO:pd==0.25.3
2021-02-16 00:39:49,073:INFO:numpy==1.18.1
2021-02-16 00:39:49,073:INFO:sklearn==0.22.1
2021-02-16 00:39:51,626:INFO:xgboost==0.90
2021-02-16 00:39:51,626:INFO:lightgbm==3.1.1
2021-02-16 00:39:54,153:INFO:catboost==0.24.4
2021-02-16 00:40:04,799:INFO:mlflow==1.13.1
2021-02-16 00:40:04,800:INFO:Checking Exceptions
2021-02-16 00:40:04,800:INFO:Declaring global variables
2021-02-16 00:40:04,800:INFO:USI: 0cb6
2021-02-16 00:40:04,800:INFO:pycaret_globals: {'transform_target_param', 'stratify_param', 'fix_imbalance_method_param', 'X_train', 'prep_pipe', '_gpu_n_jobs_param', 'experiment__', 'fold_groups_param', 'n_jobs_param', 'display_container', 'fix_imbalance_param', 'USI', 'log_plots_param', '_internal_pipeline', 'y', 'X', 'y_train', '_all_models_internal', 'target_param', '_available_plots', 'pycaret_globals', '_all_models', 'X_test', '_all_metrics', 'transform_target_method_param', 'create_model_container', 'exp_name_log', '_ml_usecase', 'imputation_classifier', 'y_test', 'logging_param', 'html_param', 'seed', 'gpu_param', 'fold_generator', 'fold_param', 'imputation_regressor', 'fold_shuffle_param', 'data_before_preprocess', 'iterative_imputation_iters_param', 'master_model_container'}
2021-02-16 00:40:04,800:INFO:Preparing display monitor
2021-02-16 00:40:04,801:INFO:Preparing display monitor
2021-02-16 00:40:04,826:INFO:Importing libraries
2021-02-16 00:40:04,826:INFO:Copying data for preprocessing
2021-02-16 00:40:04,843:INFO:Declaring preprocessing parameters
2021-02-16 00:40:04,848:INFO:Creating preprocessing pipeline
2021-02-16 00:40:04,883:INFO:Preprocessing pipeline created successfully
2021-02-16 00:40:04,883:ERROR:(Process Exit): setup has been interupted with user command 'quit'. setup must rerun.
2021-02-16 00:40:04,883:INFO:Creating global containers
2021-02-16 00:40:04,971:INFO:Internal pipeline: Pipeline(memory=None, steps=[('empty_step', 'passthrough')], verbose=False)
2021-02-16 00:48:19,917:INFO:Creating grid variables
2021-02-16 00:48:20,005:INFO:create_model_container: 0
2021-02-16 00:48:20,005:INFO:master_model_container: 0
2021-02-16 00:48:20,005:INFO:display_container: 1
2021-02-16 00:48:20,012:INFO:Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      id_columns=[], ml_usecase='regression',
                                      numerical_features=[], target='count_log',
                                      time_features=[])),
                ('imputer',
                 Simple_Imputer(categorical_strategy='not_available',
                                fill_value_categorical=None,
                                fill_value_numerical=None,
                                numeric_strate...
                ('scaling', 'passthrough'), ('P_transform', 'passthrough'),
                ('binn', 'passthrough'), ('rem_outliers', 'passthrough'),
                ('cluster_all', 'passthrough'),
                ('dummy', Dummify(target='count_log')),
                ('fix_perfect', Remove_100(target='count_log')),
                ('clean_names', Clean_Colum_Names()),
                ('feature_select', 'passthrough'), ('fix_multi', 'passthrough'),
                ('dfs', 'passthrough'), ('pca', 'passthrough')],
         verbose=False)
2021-02-16 00:48:20,013:INFO:setup() succesfully completed......................................
2021-02-16 00:50:05,963:INFO:PyCaret Supervised Module
2021-02-16 00:50:05,963:INFO:ML Usecase: regression
2021-02-16 00:50:05,963:INFO:version 2.2.3
2021-02-16 00:50:05,964:INFO:Initializing setup()
2021-02-16 00:50:05,964:INFO:setup(target=count_log, ml_usecase=regression, available_plots={'parameter': 'Hyperparameters', 'residuals': 'Residuals', 'error': 'Prediction Error', 'cooks': 'Cooks Distance', 'rfe': 'Feature Selection', 'learning': 'Learning Curve', 'manifold': 'Manifold Learning', 'vc': 'Validation Curve', 'feature': 'Feature Importance', 'feature_all': 'Feature Importance (All)', 'tree': 'Decision Tree'}, train_size=0.7, test_data=None, preprocess=True, imputation_type=simple, iterative_imputation_iters=5, categorical_features=None, categorical_imputation=constant, categorical_iterative_imputer=lightgbm, ordinal_features=None, high_cardinality_features=None, high_cardinality_method=frequency, numeric_features=['day', 'hour', 'humidity', 'month', 'temp', 'weekend', 'windspeed_log', 'workingday', 'year', 'season_2', 'season_3', 'season_4', 'weather_2', 'weather_3', 'weather_4'], numeric_imputation=mean, numeric_iterative_imputer=lightgbm, date_features=None, ignore_features=None, normalize=False, normalize_method=zscore, transformation=False, transformation_method=yeo-johnson, handle_unknown_categorical=True, unknown_categorical_method=least_frequent, pca=False, pca_method=linear, pca_components=None, ignore_low_variance=False, combine_rare_levels=False, rare_level_threshold=0.1, bin_numeric_features=None, remove_outliers=False, outliers_threshold=0.05, remove_multicollinearity=False, multicollinearity_threshold=0.9, remove_perfect_collinearity=True, create_clusters=False, cluster_iter=20, polynomial_features=False, polynomial_degree=2, trigonometry_features=False, polynomial_threshold=0.1, group_features=None, group_names=None, feature_selection=False, feature_selection_threshold=0.8, feature_selection_method=classic, feature_interaction=False, feature_ratio=False, interaction_threshold=0.01, fix_imbalance=False, fix_imbalance_method=None, transform_target=False, transform_target_method=box-cox, data_split_shuffle=True, data_split_stratify=False, fold_strategy=kfold, fold=10, fold_shuffle=False, fold_groups=None, n_jobs=-1, use_gpu=False, custom_pipeline=None, html=True, session_id=786, log_experiment=False, experiment_name=None, log_plots=False, log_profile=False, log_data=False, silent=False, verbose=True, profile=False, profile_kwargs=None, display=None)
2021-02-16 00:50:05,964:INFO:Checking environment
2021-02-16 00:50:05,964:INFO:python_version: 3.7.4
2021-02-16 00:50:05,964:INFO:python_build: ('default', 'Aug  9 2019 18:34:13')
2021-02-16 00:50:05,964:INFO:machine: AMD64
2021-02-16 00:50:05,964:INFO:platform: Windows-10-10.0.19041-SP0
2021-02-16 00:50:05,965:INFO:Memory: svmem(total=8504934400, available=2315161600, percent=72.8, used=6189772800, free=2315161600)
2021-02-16 00:50:05,965:INFO:Physical Core: 2
2021-02-16 00:50:05,965:INFO:Logical Core: 4
2021-02-16 00:50:05,965:INFO:Checking libraries
2021-02-16 00:50:05,965:INFO:pd==0.25.3
2021-02-16 00:50:05,965:INFO:numpy==1.18.1
2021-02-16 00:50:05,965:INFO:sklearn==0.22.1
2021-02-16 00:50:05,965:INFO:xgboost==0.90
2021-02-16 00:50:05,966:INFO:lightgbm==3.1.1
2021-02-16 00:50:05,966:INFO:catboost==0.24.4
2021-02-16 00:50:05,966:INFO:mlflow==1.13.1
2021-02-16 00:50:05,966:INFO:Checking Exceptions
2021-02-16 00:50:05,966:INFO:Declaring global variables
2021-02-16 00:50:05,966:INFO:USI: 48ba
2021-02-16 00:50:05,966:INFO:pycaret_globals: {'transform_target_param', 'stratify_param', 'fix_imbalance_method_param', 'X_train', 'prep_pipe', '_gpu_n_jobs_param', 'experiment__', 'fold_groups_param', 'n_jobs_param', 'display_container', 'fix_imbalance_param', 'USI', 'log_plots_param', '_internal_pipeline', 'y', 'X', 'y_train', '_all_models_internal', 'target_param', '_available_plots', 'pycaret_globals', '_all_models', 'X_test', '_all_metrics', 'transform_target_method_param', 'create_model_container', 'exp_name_log', '_ml_usecase', 'imputation_classifier', 'y_test', 'logging_param', 'html_param', 'seed', 'gpu_param', 'fold_generator', 'fold_param', 'imputation_regressor', 'fold_shuffle_param', 'data_before_preprocess', 'iterative_imputation_iters_param', 'master_model_container'}
2021-02-16 00:50:05,966:INFO:Preparing display monitor
2021-02-16 00:50:05,967:INFO:Preparing display monitor
2021-02-16 00:50:05,985:INFO:Importing libraries
2021-02-16 00:50:05,985:INFO:Copying data for preprocessing
2021-02-16 00:50:05,999:INFO:Declaring preprocessing parameters
2021-02-16 00:50:06,004:INFO:Creating preprocessing pipeline
2021-02-16 00:50:06,012:INFO:Preprocessing pipeline created successfully
2021-02-16 00:50:06,012:ERROR:(Process Exit): setup has been interupted with user command 'quit'. setup must rerun.
2021-02-16 00:50:06,012:INFO:Creating global containers
2021-02-16 00:50:06,013:INFO:Internal pipeline: Pipeline(memory=None, steps=[('empty_step', 'passthrough')], verbose=False)
2021-02-16 00:50:18,867:INFO:Creating grid variables
2021-02-16 00:50:18,934:INFO:create_model_container: 0
2021-02-16 00:50:18,934:INFO:master_model_container: 0
2021-02-16 00:50:18,935:INFO:display_container: 1
2021-02-16 00:50:18,945:INFO:Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      id_columns=[], ml_usecase='regression',
                                      numerical_features=['day', 'hour',
                                                          'humidity', 'month',
                                                          'temp', 'weekend',
                                                          'windspeed_log',
                                                          'workingday', 'year',
                                                          'season_2',
                                                          'season_3',
                                                          'season_4',
                                                          'weather_2',
                                                          'weather_3',
                                                          'weather_4'],
                                      target='count_lo...
                ('scaling', 'passthrough'), ('P_transform', 'passthrough'),
                ('binn', 'passthrough'), ('rem_outliers', 'passthrough'),
                ('cluster_all', 'passthrough'),
                ('dummy', Dummify(target='count_log')),
                ('fix_perfect', Remove_100(target='count_log')),
                ('clean_names', Clean_Colum_Names()),
                ('feature_select', 'passthrough'), ('fix_multi', 'passthrough'),
                ('dfs', 'passthrough'), ('pca', 'passthrough')],
         verbose=False)
2021-02-16 00:50:18,945:INFO:setup() succesfully completed......................................
2021-02-16 00:50:29,489:INFO:Initializing compare_models()
2021-02-16 00:50:29,489:INFO:compare_models(include=None, fold=None, round=4, cross_validation=True, sort=R2, n_select=1, budget_time=None, turbo=True, errors=ignore, fit_kwargs=None, groups=None, verbose=True, display=None, exclude=None)
2021-02-16 00:50:29,490:INFO:Checking exceptions
2021-02-16 00:50:29,490:INFO:Preparing display monitor
2021-02-16 00:50:29,491:INFO:Preparing display monitor
2021-02-16 00:50:29,552:INFO:Initializing Linear Regression
2021-02-16 00:50:29,552:INFO:Total runtime is 0.0 minutes
2021-02-16 00:50:29,571:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:29,571:INFO:Initializing create_model()
2021-02-16 00:50:29,572:INFO:create_model(estimator=lr, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:29,572:INFO:Checking exceptions
2021-02-16 00:50:29,572:INFO:Importing libraries
2021-02-16 00:50:29,572:INFO:Copying training dataset
2021-02-16 00:50:29,574:INFO:Defining folds
2021-02-16 00:50:29,574:INFO:Declaring metric variables
2021-02-16 00:50:29,589:INFO:Importing untrained model
2021-02-16 00:50:29,603:INFO:Linear Regression Imported succesfully
2021-02-16 00:50:29,636:INFO:Starting cross validation
2021-02-16 00:50:29,727:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:48,054:INFO:Calculating mean and std
2021-02-16 00:50:48,055:INFO:Creating metrics dataframe
2021-02-16 00:50:48,063:INFO:Uploading results into container
2021-02-16 00:50:48,063:INFO:Uploading model into container now
2021-02-16 00:50:48,064:INFO:create_model_container: 1
2021-02-16 00:50:48,064:INFO:master_model_container: 1
2021-02-16 00:50:48,065:INFO:display_container: 2
2021-02-16 00:50:48,065:INFO:LinearRegression(copy_X=True, fit_intercept=True, n_jobs=-1, normalize=False)
2021-02-16 00:50:48,065:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:48,988:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:48,989:INFO:Creating metrics dataframe
2021-02-16 00:50:49,006:INFO:Initializing Lasso Regression
2021-02-16 00:50:49,006:INFO:Total runtime is 0.32423112392425535 minutes
2021-02-16 00:50:49,020:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:49,020:INFO:Initializing create_model()
2021-02-16 00:50:49,021:INFO:create_model(estimator=lasso, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:49,021:INFO:Checking exceptions
2021-02-16 00:50:49,021:INFO:Importing libraries
2021-02-16 00:50:49,021:INFO:Copying training dataset
2021-02-16 00:50:49,023:INFO:Defining folds
2021-02-16 00:50:49,023:INFO:Declaring metric variables
2021-02-16 00:50:49,036:INFO:Importing untrained model
2021-02-16 00:50:49,053:INFO:Lasso Regression Imported succesfully
2021-02-16 00:50:49,091:INFO:Starting cross validation
2021-02-16 00:50:49,092:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:49,301:INFO:Calculating mean and std
2021-02-16 00:50:49,302:INFO:Creating metrics dataframe
2021-02-16 00:50:49,309:INFO:Uploading results into container
2021-02-16 00:50:49,310:INFO:Uploading model into container now
2021-02-16 00:50:49,310:INFO:create_model_container: 2
2021-02-16 00:50:49,310:INFO:master_model_container: 2
2021-02-16 00:50:49,310:INFO:display_container: 2
2021-02-16 00:50:49,311:INFO:Lasso(alpha=1.0, copy_X=True, fit_intercept=True, max_iter=1000,
      normalize=False, positive=False, precompute=False, random_state=786,
      selection='cyclic', tol=0.0001, warm_start=False)
2021-02-16 00:50:49,311:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:50,225:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:50,225:INFO:Creating metrics dataframe
2021-02-16 00:50:50,245:INFO:Initializing Ridge Regression
2021-02-16 00:50:50,245:INFO:Total runtime is 0.34486927986145016 minutes
2021-02-16 00:50:50,257:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:50,258:INFO:Initializing create_model()
2021-02-16 00:50:50,258:INFO:create_model(estimator=ridge, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:50,259:INFO:Checking exceptions
2021-02-16 00:50:50,259:INFO:Importing libraries
2021-02-16 00:50:50,259:INFO:Copying training dataset
2021-02-16 00:50:50,261:INFO:Defining folds
2021-02-16 00:50:50,261:INFO:Declaring metric variables
2021-02-16 00:50:50,273:INFO:Importing untrained model
2021-02-16 00:50:50,286:INFO:Ridge Regression Imported succesfully
2021-02-16 00:50:50,313:INFO:Starting cross validation
2021-02-16 00:50:50,314:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:50,564:INFO:Calculating mean and std
2021-02-16 00:50:50,566:INFO:Creating metrics dataframe
2021-02-16 00:50:50,571:INFO:Uploading results into container
2021-02-16 00:50:50,571:INFO:Uploading model into container now
2021-02-16 00:50:50,571:INFO:create_model_container: 3
2021-02-16 00:50:50,572:INFO:master_model_container: 3
2021-02-16 00:50:50,572:INFO:display_container: 2
2021-02-16 00:50:50,572:INFO:Ridge(alpha=1.0, copy_X=True, fit_intercept=True, max_iter=None,
      normalize=False, random_state=786, solver='auto', tol=0.001)
2021-02-16 00:50:50,572:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:51,409:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:51,409:INFO:Creating metrics dataframe
2021-02-16 00:50:51,433:INFO:Initializing Elastic Net
2021-02-16 00:50:51,433:INFO:Total runtime is 0.3646690766016642 minutes
2021-02-16 00:50:51,446:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:51,447:INFO:Initializing create_model()
2021-02-16 00:50:51,448:INFO:create_model(estimator=en, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:51,448:INFO:Checking exceptions
2021-02-16 00:50:51,449:INFO:Importing libraries
2021-02-16 00:50:51,449:INFO:Copying training dataset
2021-02-16 00:50:51,451:INFO:Defining folds
2021-02-16 00:50:51,451:INFO:Declaring metric variables
2021-02-16 00:50:51,463:INFO:Importing untrained model
2021-02-16 00:50:51,475:INFO:Elastic Net Imported succesfully
2021-02-16 00:50:51,503:INFO:Starting cross validation
2021-02-16 00:50:51,504:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:51,625:INFO:Calculating mean and std
2021-02-16 00:50:51,627:INFO:Creating metrics dataframe
2021-02-16 00:50:51,634:INFO:Uploading results into container
2021-02-16 00:50:51,634:INFO:Uploading model into container now
2021-02-16 00:50:51,635:INFO:create_model_container: 4
2021-02-16 00:50:51,635:INFO:master_model_container: 4
2021-02-16 00:50:51,635:INFO:display_container: 2
2021-02-16 00:50:51,635:INFO:ElasticNet(alpha=1.0, copy_X=True, fit_intercept=True, l1_ratio=0.5,
           max_iter=1000, normalize=False, positive=False, precompute=False,
           random_state=786, selection='cyclic', tol=0.0001, warm_start=False)
2021-02-16 00:50:51,635:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:52,467:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:52,467:INFO:Creating metrics dataframe
2021-02-16 00:50:52,491:INFO:Initializing Least Angle Regression
2021-02-16 00:50:52,492:INFO:Total runtime is 0.3823304255803426 minutes
2021-02-16 00:50:52,505:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:52,506:INFO:Initializing create_model()
2021-02-16 00:50:52,506:INFO:create_model(estimator=lar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:52,506:INFO:Checking exceptions
2021-02-16 00:50:52,506:INFO:Importing libraries
2021-02-16 00:50:52,507:INFO:Copying training dataset
2021-02-16 00:50:52,509:INFO:Defining folds
2021-02-16 00:50:52,509:INFO:Declaring metric variables
2021-02-16 00:50:52,520:INFO:Importing untrained model
2021-02-16 00:50:52,520:WARNING:create_model() for lar raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 00:50:52,574:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 00:50:52,575:INFO:Initializing create_model()
2021-02-16 00:50:52,575:INFO:create_model(estimator=lar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:52,575:INFO:Checking exceptions
2021-02-16 00:50:52,575:INFO:Importing libraries
2021-02-16 00:50:52,575:INFO:Copying training dataset
2021-02-16 00:50:52,576:INFO:Defining folds
2021-02-16 00:50:52,577:INFO:Declaring metric variables
2021-02-16 00:50:52,593:INFO:Importing untrained model
2021-02-16 00:50:52,594:ERROR:create_model() for lar raised an exception:
2021-02-16 00:50:52,595:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 00:50:52,595:INFO:Initializing Lasso Least Angle Regression
2021-02-16 00:50:52,595:INFO:Total runtime is 0.3840365330378214 minutes
2021-02-16 00:50:52,617:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:52,618:INFO:Initializing create_model()
2021-02-16 00:50:52,618:INFO:create_model(estimator=llar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:52,618:INFO:Checking exceptions
2021-02-16 00:50:52,619:INFO:Importing libraries
2021-02-16 00:50:52,619:INFO:Copying training dataset
2021-02-16 00:50:52,621:INFO:Defining folds
2021-02-16 00:50:52,622:INFO:Declaring metric variables
2021-02-16 00:50:52,639:INFO:Importing untrained model
2021-02-16 00:50:52,639:WARNING:create_model() for llar raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 00:50:52,640:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 00:50:52,640:INFO:Initializing create_model()
2021-02-16 00:50:52,641:INFO:create_model(estimator=llar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:52,641:INFO:Checking exceptions
2021-02-16 00:50:52,641:INFO:Importing libraries
2021-02-16 00:50:52,641:INFO:Copying training dataset
2021-02-16 00:50:52,643:INFO:Defining folds
2021-02-16 00:50:52,643:INFO:Declaring metric variables
2021-02-16 00:50:52,658:INFO:Importing untrained model
2021-02-16 00:50:52,658:ERROR:create_model() for llar raised an exception:
2021-02-16 00:50:52,659:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 00:50:52,660:INFO:Initializing Orthogonal Matching Pursuit
2021-02-16 00:50:52,660:INFO:Total runtime is 0.38511922756830846 minutes
2021-02-16 00:50:52,676:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:52,677:INFO:Initializing create_model()
2021-02-16 00:50:52,677:INFO:create_model(estimator=omp, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:52,677:INFO:Checking exceptions
2021-02-16 00:50:52,678:INFO:Importing libraries
2021-02-16 00:50:52,678:INFO:Copying training dataset
2021-02-16 00:50:52,681:INFO:Defining folds
2021-02-16 00:50:52,682:INFO:Declaring metric variables
2021-02-16 00:50:52,698:INFO:Importing untrained model
2021-02-16 00:50:52,714:INFO:Orthogonal Matching Pursuit Imported succesfully
2021-02-16 00:50:52,747:INFO:Starting cross validation
2021-02-16 00:50:52,748:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:52,896:INFO:Calculating mean and std
2021-02-16 00:50:52,898:INFO:Creating metrics dataframe
2021-02-16 00:50:52,907:INFO:Uploading results into container
2021-02-16 00:50:52,907:INFO:Uploading model into container now
2021-02-16 00:50:52,907:INFO:create_model_container: 5
2021-02-16 00:50:52,907:INFO:master_model_container: 5
2021-02-16 00:50:52,908:INFO:display_container: 2
2021-02-16 00:50:52,908:INFO:OrthogonalMatchingPursuit(fit_intercept=True, n_nonzero_coefs=None,
                          normalize=True, precompute='auto', tol=None)
2021-02-16 00:50:52,908:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:53,805:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:53,805:INFO:Creating metrics dataframe
2021-02-16 00:50:53,845:INFO:Initializing Bayesian Ridge
2021-02-16 00:50:53,845:INFO:Total runtime is 0.4048745473225911 minutes
2021-02-16 00:50:53,859:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:53,860:INFO:Initializing create_model()
2021-02-16 00:50:53,860:INFO:create_model(estimator=br, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:53,861:INFO:Checking exceptions
2021-02-16 00:50:53,861:INFO:Importing libraries
2021-02-16 00:50:53,861:INFO:Copying training dataset
2021-02-16 00:50:53,863:INFO:Defining folds
2021-02-16 00:50:53,863:INFO:Declaring metric variables
2021-02-16 00:50:53,875:INFO:Importing untrained model
2021-02-16 00:50:53,886:INFO:Bayesian Ridge Imported succesfully
2021-02-16 00:50:53,911:INFO:Starting cross validation
2021-02-16 00:50:53,912:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:54,061:INFO:Calculating mean and std
2021-02-16 00:50:54,063:INFO:Creating metrics dataframe
2021-02-16 00:50:54,072:INFO:Uploading results into container
2021-02-16 00:50:54,072:INFO:Uploading model into container now
2021-02-16 00:50:54,073:INFO:create_model_container: 6
2021-02-16 00:50:54,073:INFO:master_model_container: 6
2021-02-16 00:50:54,073:INFO:display_container: 2
2021-02-16 00:50:54,074:INFO:BayesianRidge(alpha_1=1e-06, alpha_2=1e-06, alpha_init=None,
              compute_score=False, copy_X=True, fit_intercept=True,
              lambda_1=1e-06, lambda_2=1e-06, lambda_init=None, n_iter=300,
              normalize=False, tol=0.001, verbose=False)
2021-02-16 00:50:54,074:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:55,018:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:55,018:INFO:Creating metrics dataframe
2021-02-16 00:50:55,051:INFO:Initializing Passive Aggressive Regressor
2021-02-16 00:50:55,052:INFO:Total runtime is 0.4249963561693827 minutes
2021-02-16 00:50:55,068:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:55,068:INFO:Initializing create_model()
2021-02-16 00:50:55,068:INFO:create_model(estimator=par, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:55,069:INFO:Checking exceptions
2021-02-16 00:50:55,069:INFO:Importing libraries
2021-02-16 00:50:55,069:INFO:Copying training dataset
2021-02-16 00:50:55,070:INFO:Defining folds
2021-02-16 00:50:55,070:INFO:Declaring metric variables
2021-02-16 00:50:55,083:INFO:Importing untrained model
2021-02-16 00:50:55,098:INFO:Passive Aggressive Regressor Imported succesfully
2021-02-16 00:50:55,123:INFO:Starting cross validation
2021-02-16 00:50:55,125:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:55,376:INFO:Calculating mean and std
2021-02-16 00:50:55,379:INFO:Creating metrics dataframe
2021-02-16 00:50:55,386:INFO:Uploading results into container
2021-02-16 00:50:55,386:INFO:Uploading model into container now
2021-02-16 00:50:55,387:INFO:create_model_container: 7
2021-02-16 00:50:55,387:INFO:master_model_container: 7
2021-02-16 00:50:55,387:INFO:display_container: 2
2021-02-16 00:50:55,387:INFO:PassiveAggressiveRegressor(C=1.0, average=False, early_stopping=False,
                           epsilon=0.1, fit_intercept=True,
                           loss='epsilon_insensitive', max_iter=1000,
                           n_iter_no_change=5, random_state=786, shuffle=True,
                           tol=0.001, validation_fraction=0.1, verbose=0,
                           warm_start=False)
2021-02-16 00:50:55,387:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:56,435:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:56,435:INFO:Creating metrics dataframe
2021-02-16 00:50:56,467:INFO:Initializing Huber Regressor
2021-02-16 00:50:56,468:INFO:Total runtime is 0.44859073162078855 minutes
2021-02-16 00:50:56,483:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:56,483:INFO:Initializing create_model()
2021-02-16 00:50:56,484:INFO:create_model(estimator=huber, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:56,484:INFO:Checking exceptions
2021-02-16 00:50:56,484:INFO:Importing libraries
2021-02-16 00:50:56,484:INFO:Copying training dataset
2021-02-16 00:50:56,485:INFO:Defining folds
2021-02-16 00:50:56,486:INFO:Declaring metric variables
2021-02-16 00:50:56,497:INFO:Importing untrained model
2021-02-16 00:50:56,511:INFO:Huber Regressor Imported succesfully
2021-02-16 00:50:56,542:INFO:Starting cross validation
2021-02-16 00:50:56,543:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:57,804:INFO:Calculating mean and std
2021-02-16 00:50:57,806:INFO:Creating metrics dataframe
2021-02-16 00:50:57,811:INFO:Uploading results into container
2021-02-16 00:50:57,812:INFO:Uploading model into container now
2021-02-16 00:50:57,812:INFO:create_model_container: 8
2021-02-16 00:50:57,812:INFO:master_model_container: 8
2021-02-16 00:50:57,812:INFO:display_container: 2
2021-02-16 00:50:57,813:INFO:HuberRegressor(alpha=0.0001, epsilon=1.35, fit_intercept=True, max_iter=100,
               tol=1e-05, warm_start=False)
2021-02-16 00:50:57,813:INFO:create_model() succesfully completed......................................
2021-02-16 00:50:58,696:INFO:SubProcess create_model() end ==================================
2021-02-16 00:50:58,696:INFO:Creating metrics dataframe
2021-02-16 00:50:58,731:INFO:Initializing K Neighbors Regressor
2021-02-16 00:50:58,731:INFO:Total runtime is 0.48630061944325764 minutes
2021-02-16 00:50:58,744:INFO:SubProcess create_model() called ==================================
2021-02-16 00:50:58,744:INFO:Initializing create_model()
2021-02-16 00:50:58,744:INFO:create_model(estimator=knn, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:50:58,744:INFO:Checking exceptions
2021-02-16 00:50:58,746:INFO:Importing libraries
2021-02-16 00:50:58,746:INFO:Copying training dataset
2021-02-16 00:50:58,747:INFO:Defining folds
2021-02-16 00:50:58,747:INFO:Declaring metric variables
2021-02-16 00:50:58,759:INFO:Importing untrained model
2021-02-16 00:50:58,771:INFO:K Neighbors Regressor Imported succesfully
2021-02-16 00:50:58,800:INFO:Starting cross validation
2021-02-16 00:50:58,801:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:50:59,539:INFO:Calculating mean and std
2021-02-16 00:50:59,540:INFO:Creating metrics dataframe
2021-02-16 00:50:59,555:INFO:Uploading results into container
2021-02-16 00:50:59,556:INFO:Uploading model into container now
2021-02-16 00:50:59,557:INFO:create_model_container: 9
2021-02-16 00:50:59,557:INFO:master_model_container: 9
2021-02-16 00:50:59,558:INFO:display_container: 2
2021-02-16 00:50:59,559:INFO:KNeighborsRegressor(algorithm='auto', leaf_size=30, metric='minkowski',
                    metric_params=None, n_jobs=-1, n_neighbors=5, p=2,
                    weights='uniform')
2021-02-16 00:50:59,559:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:00,412:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:00,413:INFO:Creating metrics dataframe
2021-02-16 00:51:00,451:INFO:Initializing Decision Tree Regressor
2021-02-16 00:51:00,452:INFO:Total runtime is 0.5149943073590596 minutes
2021-02-16 00:51:00,465:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:00,466:INFO:Initializing create_model()
2021-02-16 00:51:00,466:INFO:create_model(estimator=dt, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:00,466:INFO:Checking exceptions
2021-02-16 00:51:00,466:INFO:Importing libraries
2021-02-16 00:51:00,467:INFO:Copying training dataset
2021-02-16 00:51:00,468:INFO:Defining folds
2021-02-16 00:51:00,469:INFO:Declaring metric variables
2021-02-16 00:51:00,480:INFO:Importing untrained model
2021-02-16 00:51:00,494:INFO:Decision Tree Regressor Imported succesfully
2021-02-16 00:51:00,522:INFO:Starting cross validation
2021-02-16 00:51:00,523:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:00,911:INFO:Calculating mean and std
2021-02-16 00:51:00,913:INFO:Creating metrics dataframe
2021-02-16 00:51:00,919:INFO:Uploading results into container
2021-02-16 00:51:00,919:INFO:Uploading model into container now
2021-02-16 00:51:00,920:INFO:create_model_container: 10
2021-02-16 00:51:00,920:INFO:master_model_container: 10
2021-02-16 00:51:00,920:INFO:display_container: 2
2021-02-16 00:51:00,921:INFO:DecisionTreeRegressor(ccp_alpha=0.0, criterion='mse', max_depth=None,
                      max_features=None, max_leaf_nodes=None,
                      min_impurity_decrease=0.0, min_impurity_split=None,
                      min_samples_leaf=1, min_samples_split=2,
                      min_weight_fraction_leaf=0.0, presort='deprecated',
                      random_state=786, splitter='best')
2021-02-16 00:51:00,921:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:01,916:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:01,917:INFO:Creating metrics dataframe
2021-02-16 00:51:01,971:INFO:Initializing Random Forest Regressor
2021-02-16 00:51:01,971:INFO:Total runtime is 0.540303607781728 minutes
2021-02-16 00:51:01,990:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:01,991:INFO:Initializing create_model()
2021-02-16 00:51:01,992:INFO:create_model(estimator=rf, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:01,992:INFO:Checking exceptions
2021-02-16 00:51:01,992:INFO:Importing libraries
2021-02-16 00:51:01,992:INFO:Copying training dataset
2021-02-16 00:51:01,994:INFO:Defining folds
2021-02-16 00:51:01,994:INFO:Declaring metric variables
2021-02-16 00:51:02,009:INFO:Importing untrained model
2021-02-16 00:51:02,028:INFO:Random Forest Regressor Imported succesfully
2021-02-16 00:51:02,062:INFO:Starting cross validation
2021-02-16 00:51:02,065:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:15,868:INFO:Calculating mean and std
2021-02-16 00:51:15,870:INFO:Creating metrics dataframe
2021-02-16 00:51:15,881:INFO:Uploading results into container
2021-02-16 00:51:15,882:INFO:Uploading model into container now
2021-02-16 00:51:15,882:INFO:create_model_container: 11
2021-02-16 00:51:15,882:INFO:master_model_container: 11
2021-02-16 00:51:15,882:INFO:display_container: 2
2021-02-16 00:51:15,884:INFO:RandomForestRegressor(bootstrap=True, ccp_alpha=0.0, criterion='mse',
                      max_depth=None, max_features='auto', max_leaf_nodes=None,
                      max_samples=None, min_impurity_decrease=0.0,
                      min_impurity_split=None, min_samples_leaf=1,
                      min_samples_split=2, min_weight_fraction_leaf=0.0,
                      n_estimators=100, n_jobs=-1, oob_score=False,
                      random_state=786, verbose=0, warm_start=False)
2021-02-16 00:51:15,884:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:17,070:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:17,071:INFO:Creating metrics dataframe
2021-02-16 00:51:17,114:INFO:Initializing Extra Trees Regressor
2021-02-16 00:51:17,115:INFO:Total runtime is 0.7927050193150837 minutes
2021-02-16 00:51:17,129:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:17,130:INFO:Initializing create_model()
2021-02-16 00:51:17,130:INFO:create_model(estimator=et, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:17,130:INFO:Checking exceptions
2021-02-16 00:51:17,131:INFO:Importing libraries
2021-02-16 00:51:17,131:INFO:Copying training dataset
2021-02-16 00:51:17,133:INFO:Defining folds
2021-02-16 00:51:17,133:INFO:Declaring metric variables
2021-02-16 00:51:17,149:INFO:Importing untrained model
2021-02-16 00:51:17,167:INFO:Extra Trees Regressor Imported succesfully
2021-02-16 00:51:17,205:INFO:Starting cross validation
2021-02-16 00:51:17,205:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:30,125:INFO:Calculating mean and std
2021-02-16 00:51:30,132:INFO:Creating metrics dataframe
2021-02-16 00:51:30,149:INFO:Uploading results into container
2021-02-16 00:51:30,150:INFO:Uploading model into container now
2021-02-16 00:51:30,152:INFO:create_model_container: 12
2021-02-16 00:51:30,153:INFO:master_model_container: 12
2021-02-16 00:51:30,153:INFO:display_container: 2
2021-02-16 00:51:30,157:INFO:ExtraTreesRegressor(bootstrap=False, ccp_alpha=0.0, criterion='mse',
                    max_depth=None, max_features='auto', max_leaf_nodes=None,
                    max_samples=None, min_impurity_decrease=0.0,
                    min_impurity_split=None, min_samples_leaf=1,
                    min_samples_split=2, min_weight_fraction_leaf=0.0,
                    n_estimators=100, n_jobs=-1, oob_score=False,
                    random_state=786, verbose=0, warm_start=False)
2021-02-16 00:51:30,157:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:31,174:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:31,175:INFO:Creating metrics dataframe
2021-02-16 00:51:31,223:INFO:Initializing AdaBoost Regressor
2021-02-16 00:51:31,223:INFO:Total runtime is 1.0278373638788858 minutes
2021-02-16 00:51:31,238:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:31,239:INFO:Initializing create_model()
2021-02-16 00:51:31,239:INFO:create_model(estimator=ada, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:31,239:INFO:Checking exceptions
2021-02-16 00:51:31,240:INFO:Importing libraries
2021-02-16 00:51:31,240:INFO:Copying training dataset
2021-02-16 00:51:31,241:INFO:Defining folds
2021-02-16 00:51:31,242:INFO:Declaring metric variables
2021-02-16 00:51:31,254:INFO:Importing untrained model
2021-02-16 00:51:31,273:INFO:AdaBoost Regressor Imported succesfully
2021-02-16 00:51:31,307:INFO:Starting cross validation
2021-02-16 00:51:31,308:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:34,145:INFO:Calculating mean and std
2021-02-16 00:51:34,146:INFO:Creating metrics dataframe
2021-02-16 00:51:34,152:INFO:Uploading results into container
2021-02-16 00:51:34,152:INFO:Uploading model into container now
2021-02-16 00:51:34,152:INFO:create_model_container: 13
2021-02-16 00:51:34,152:INFO:master_model_container: 13
2021-02-16 00:51:34,152:INFO:display_container: 2
2021-02-16 00:51:34,152:INFO:AdaBoostRegressor(base_estimator=None, learning_rate=1.0, loss='linear',
                  n_estimators=50, random_state=786)
2021-02-16 00:51:34,152:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:34,999:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:34,999:INFO:Creating metrics dataframe
2021-02-16 00:51:35,051:INFO:Initializing Gradient Boosting Regressor
2021-02-16 00:51:35,052:INFO:Total runtime is 1.0916586875915526 minutes
2021-02-16 00:51:35,064:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:35,065:INFO:Initializing create_model()
2021-02-16 00:51:35,065:INFO:create_model(estimator=gbr, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:35,065:INFO:Checking exceptions
2021-02-16 00:51:35,066:INFO:Importing libraries
2021-02-16 00:51:35,066:INFO:Copying training dataset
2021-02-16 00:51:35,067:INFO:Defining folds
2021-02-16 00:51:35,067:INFO:Declaring metric variables
2021-02-16 00:51:35,082:INFO:Importing untrained model
2021-02-16 00:51:35,096:INFO:Gradient Boosting Regressor Imported succesfully
2021-02-16 00:51:35,128:INFO:Starting cross validation
2021-02-16 00:51:35,129:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:38,537:INFO:Calculating mean and std
2021-02-16 00:51:38,538:INFO:Creating metrics dataframe
2021-02-16 00:51:38,545:INFO:Uploading results into container
2021-02-16 00:51:38,545:INFO:Uploading model into container now
2021-02-16 00:51:38,545:INFO:create_model_container: 14
2021-02-16 00:51:38,546:INFO:master_model_container: 14
2021-02-16 00:51:38,546:INFO:display_container: 2
2021-02-16 00:51:38,548:INFO:GradientBoostingRegressor(alpha=0.9, ccp_alpha=0.0, criterion='friedman_mse',
                          init=None, learning_rate=0.1, loss='ls', max_depth=3,
                          max_features=None, max_leaf_nodes=None,
                          min_impurity_decrease=0.0, min_impurity_split=None,
                          min_samples_leaf=1, min_samples_split=2,
                          min_weight_fraction_leaf=0.0, n_estimators=100,
                          n_iter_no_change=None, presort='deprecated',
                          random_state=786, subsample=1.0, tol=0.0001,
                          validation_fraction=0.1, verbose=0, warm_start=False)
2021-02-16 00:51:38,548:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:39,593:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:39,593:INFO:Creating metrics dataframe
2021-02-16 00:51:39,642:INFO:Initializing Extreme Gradient Boosting
2021-02-16 00:51:39,642:INFO:Total runtime is 1.1681547919909159 minutes
2021-02-16 00:51:39,657:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:39,657:INFO:Initializing create_model()
2021-02-16 00:51:39,657:INFO:create_model(estimator=xgboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:39,657:INFO:Checking exceptions
2021-02-16 00:51:39,658:INFO:Importing libraries
2021-02-16 00:51:39,658:INFO:Copying training dataset
2021-02-16 00:51:39,659:INFO:Defining folds
2021-02-16 00:51:39,659:INFO:Declaring metric variables
2021-02-16 00:51:39,672:INFO:Importing untrained model
2021-02-16 00:51:39,685:INFO:Extreme Gradient Boosting Imported succesfully
2021-02-16 00:51:39,721:INFO:Starting cross validation
2021-02-16 00:51:39,722:WARNING:create_model() for xgboost raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 00:51:39,807:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

2021-02-16 00:51:39,807:INFO:Initializing create_model()
2021-02-16 00:51:39,808:INFO:create_model(estimator=xgboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:39,808:INFO:Checking exceptions
2021-02-16 00:51:39,808:INFO:Importing libraries
2021-02-16 00:51:39,808:INFO:Copying training dataset
2021-02-16 00:51:39,809:INFO:Defining folds
2021-02-16 00:51:39,809:INFO:Declaring metric variables
2021-02-16 00:51:39,821:INFO:Importing untrained model
2021-02-16 00:51:39,838:INFO:Extreme Gradient Boosting Imported succesfully
2021-02-16 00:51:39,883:INFO:Starting cross validation
2021-02-16 00:51:39,884:ERROR:create_model() for xgboost raised an exception:
2021-02-16 00:51:39,886:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

2021-02-16 00:51:39,887:INFO:Initializing Light Gradient Boosting Machine
2021-02-16 00:51:39,887:INFO:Total runtime is 1.1722504138946532 minutes
2021-02-16 00:51:39,911:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:39,912:INFO:Initializing create_model()
2021-02-16 00:51:39,912:INFO:create_model(estimator=lightgbm, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:39,912:INFO:Checking exceptions
2021-02-16 00:51:39,912:INFO:Importing libraries
2021-02-16 00:51:39,912:INFO:Copying training dataset
2021-02-16 00:51:39,915:INFO:Defining folds
2021-02-16 00:51:39,915:INFO:Declaring metric variables
2021-02-16 00:51:39,934:INFO:Importing untrained model
2021-02-16 00:51:39,954:INFO:Light Gradient Boosting Machine Imported succesfully
2021-02-16 00:51:39,991:INFO:Starting cross validation
2021-02-16 00:51:39,992:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:51:41,215:INFO:Calculating mean and std
2021-02-16 00:51:41,216:INFO:Creating metrics dataframe
2021-02-16 00:51:41,223:INFO:Uploading results into container
2021-02-16 00:51:41,223:INFO:Uploading model into container now
2021-02-16 00:51:41,223:INFO:create_model_container: 15
2021-02-16 00:51:41,224:INFO:master_model_container: 15
2021-02-16 00:51:41,224:INFO:display_container: 2
2021-02-16 00:51:41,224:INFO:LGBMRegressor(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
              importance_type='split', learning_rate=0.1, max_depth=-1,
              min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
              n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
              random_state=786, reg_alpha=0.0, reg_lambda=0.0, silent=True,
              subsample=1.0, subsample_for_bin=200000, subsample_freq=0)
2021-02-16 00:51:41,224:INFO:create_model() succesfully completed......................................
2021-02-16 00:51:42,166:INFO:SubProcess create_model() end ==================================
2021-02-16 00:51:42,166:INFO:Creating metrics dataframe
2021-02-16 00:51:42,230:INFO:Initializing CatBoost Regressor
2021-02-16 00:51:42,230:INFO:Total runtime is 1.2112942417462667 minutes
2021-02-16 00:51:42,244:INFO:SubProcess create_model() called ==================================
2021-02-16 00:51:42,245:INFO:Initializing create_model()
2021-02-16 00:51:42,245:INFO:create_model(estimator=catboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:51:42,245:INFO:Checking exceptions
2021-02-16 00:51:42,245:INFO:Importing libraries
2021-02-16 00:51:42,246:INFO:Copying training dataset
2021-02-16 00:51:42,247:INFO:Defining folds
2021-02-16 00:51:42,247:INFO:Declaring metric variables
2021-02-16 00:51:42,259:INFO:Importing untrained model
2021-02-16 00:51:42,331:INFO:CatBoost Regressor Imported succesfully
2021-02-16 00:51:42,364:INFO:Starting cross validation
2021-02-16 00:51:42,368:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:52:24,276:INFO:Calculating mean and std
2021-02-16 00:52:24,277:INFO:Creating metrics dataframe
2021-02-16 00:52:24,283:INFO:Uploading results into container
2021-02-16 00:52:24,283:INFO:Uploading model into container now
2021-02-16 00:52:24,283:INFO:create_model_container: 16
2021-02-16 00:52:24,284:INFO:master_model_container: 16
2021-02-16 00:52:24,284:INFO:display_container: 2
2021-02-16 00:52:24,284:INFO:<catboost.core.CatBoostRegressor object at 0x000001E24D8BD148>
2021-02-16 00:52:24,284:INFO:create_model() succesfully completed......................................
2021-02-16 00:52:25,299:WARNING:create_model() for <catboost.core.CatBoostRegressor object at 0x000001E24D8BD148> raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 00:52:25,300:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2154, in compare_models
    assert np.sum(model_results.iloc[0]) != 0.0
AssertionError

2021-02-16 00:52:25,300:INFO:Initializing create_model()
2021-02-16 00:52:25,300:INFO:create_model(estimator=<catboost.core.CatBoostRegressor object at 0x000001E24D8BD148>, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x000001E252DDE708>, kwargs={})
2021-02-16 00:52:25,300:INFO:Checking exceptions
2021-02-16 00:52:25,300:INFO:Importing libraries
2021-02-16 00:52:25,300:INFO:Copying training dataset
2021-02-16 00:52:25,301:INFO:Defining folds
2021-02-16 00:52:25,302:INFO:Declaring metric variables
2021-02-16 00:52:25,313:INFO:Importing untrained model
2021-02-16 00:52:25,313:INFO:Declaring custom model
2021-02-16 00:52:25,330:INFO:CatBoost Regressor Imported succesfully
2021-02-16 00:52:25,369:INFO:Starting cross validation
2021-02-16 00:52:25,371:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 00:53:24,575:INFO:Calculating mean and std
2021-02-16 00:53:24,577:INFO:Creating metrics dataframe
2021-02-16 00:53:24,584:INFO:Uploading results into container
2021-02-16 00:53:24,584:INFO:Uploading model into container now
2021-02-16 00:53:24,584:INFO:create_model_container: 17
2021-02-16 00:53:24,584:INFO:master_model_container: 17
2021-02-16 00:53:24,585:INFO:display_container: 2
2021-02-16 00:53:24,585:INFO:<catboost.core.CatBoostRegressor object at 0x000001E252D16C88>
2021-02-16 00:53:24,585:INFO:create_model() succesfully completed......................................
2021-02-16 00:53:25,667:INFO:SubProcess create_model() end ==================================
2021-02-16 00:53:25,667:INFO:Creating metrics dataframe
2021-02-16 00:53:25,767:INFO:Initializing create_model()
2021-02-16 00:53:25,768:INFO:create_model(estimator=<catboost.core.CatBoostRegressor object at 0x000001E252D16C88>, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, verbose=False, system=False, metrics=None, display=None, kwargs={})
2021-02-16 00:53:25,768:INFO:Checking exceptions
2021-02-16 00:53:25,768:INFO:Importing libraries
2021-02-16 00:53:25,768:INFO:Copying training dataset
2021-02-16 00:53:25,770:INFO:Defining folds
2021-02-16 00:53:25,770:INFO:Declaring metric variables
2021-02-16 00:53:25,770:INFO:Importing untrained model
2021-02-16 00:53:25,770:INFO:Declaring custom model
2021-02-16 00:53:25,771:INFO:CatBoost Regressor Imported succesfully
2021-02-16 00:53:25,772:INFO:Cross validation set to False
2021-02-16 00:53:25,772:INFO:Fitting Model
2021-02-16 00:53:30,378:INFO:<catboost.core.CatBoostRegressor object at 0x000001E252B738C8>
2021-02-16 00:53:30,378:INFO:create_models() succesfully completed......................................
2021-02-16 00:53:31,422:INFO:create_model_container: 17
2021-02-16 00:53:31,422:INFO:master_model_container: 17
2021-02-16 00:53:31,422:INFO:display_container: 2
2021-02-16 00:53:31,423:INFO:<catboost.core.CatBoostRegressor object at 0x000001E252B738C8>
2021-02-16 00:53:31,423:INFO:compare_models() succesfully completed......................................
2021-02-16 01:09:48,255:INFO:Initializing create_model()
2021-02-16 01:09:48,256:INFO:create_model(estimator=rf, fold=None, round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=True, verbose=True, system=True, metrics=None, display=None, kwargs={})
2021-02-16 01:09:48,256:INFO:Checking exceptions
2021-02-16 01:09:48,256:INFO:Preparing display monitor
2021-02-16 01:09:48,291:INFO:Importing libraries
2021-02-16 01:09:48,291:INFO:Copying training dataset
2021-02-16 01:09:48,293:INFO:Defining folds
2021-02-16 01:09:48,293:INFO:Declaring metric variables
2021-02-16 01:09:48,307:INFO:Importing untrained model
2021-02-16 01:09:48,321:INFO:Random Forest Regressor Imported succesfully
2021-02-16 01:09:48,350:INFO:Starting cross validation
2021-02-16 01:09:48,351:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 01:10:06,823:INFO:Calculating mean and std
2021-02-16 01:10:06,824:INFO:Creating metrics dataframe
2021-02-16 01:10:06,847:INFO:Finalizing model
2021-02-16 01:10:08,738:INFO:Uploading results into container
2021-02-16 01:10:08,738:INFO:Uploading model into container now
2021-02-16 01:10:08,812:INFO:create_model_container: 18
2021-02-16 01:10:08,812:INFO:master_model_container: 18
2021-02-16 01:10:08,812:INFO:display_container: 3
2021-02-16 01:10:08,813:INFO:RandomForestRegressor(bootstrap=True, ccp_alpha=0.0, criterion='mse',
                      max_depth=None, max_features='auto', max_leaf_nodes=None,
                      max_samples=None, min_impurity_decrease=0.0,
                      min_impurity_split=None, min_samples_leaf=1,
                      min_samples_split=2, min_weight_fraction_leaf=0.0,
                      n_estimators=100, n_jobs=-1, oob_score=False,
                      random_state=786, verbose=0, warm_start=False)
2021-02-16 01:10:08,813:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:33,549:INFO:PyCaret Supervised Module
2021-02-16 10:29:33,550:INFO:ML Usecase: regression
2021-02-16 10:29:33,550:INFO:version 2.2.3
2021-02-16 10:29:33,550:INFO:Initializing setup()
2021-02-16 10:29:33,550:INFO:setup(target=count_log, ml_usecase=regression, available_plots={'parameter': 'Hyperparameters', 'residuals': 'Residuals', 'error': 'Prediction Error', 'cooks': 'Cooks Distance', 'rfe': 'Feature Selection', 'learning': 'Learning Curve', 'manifold': 'Manifold Learning', 'vc': 'Validation Curve', 'feature': 'Feature Importance', 'feature_all': 'Feature Importance (All)', 'tree': 'Decision Tree'}, train_size=0.7, test_data=None, preprocess=True, imputation_type=simple, iterative_imputation_iters=5, categorical_features=None, categorical_imputation=constant, categorical_iterative_imputer=lightgbm, ordinal_features=None, high_cardinality_features=None, high_cardinality_method=frequency, numeric_features=['day', 'hour', 'humidity', 'month', 'temp', 'weekend', 'windspeed_log', 'workingday', 'year', 'season_2', 'season_3', 'season_4', 'weather_2', 'weather_3', 'weather_4'], numeric_imputation=mean, numeric_iterative_imputer=lightgbm, date_features=None, ignore_features=None, normalize=False, normalize_method=zscore, transformation=False, transformation_method=yeo-johnson, handle_unknown_categorical=True, unknown_categorical_method=least_frequent, pca=False, pca_method=linear, pca_components=None, ignore_low_variance=False, combine_rare_levels=False, rare_level_threshold=0.1, bin_numeric_features=None, remove_outliers=False, outliers_threshold=0.05, remove_multicollinearity=False, multicollinearity_threshold=0.9, remove_perfect_collinearity=True, create_clusters=False, cluster_iter=20, polynomial_features=False, polynomial_degree=2, trigonometry_features=False, polynomial_threshold=0.1, group_features=None, group_names=None, feature_selection=False, feature_selection_threshold=0.8, feature_selection_method=classic, feature_interaction=False, feature_ratio=False, interaction_threshold=0.01, fix_imbalance=False, fix_imbalance_method=None, transform_target=False, transform_target_method=box-cox, data_split_shuffle=True, data_split_stratify=False, fold_strategy=kfold, fold=10, fold_shuffle=False, fold_groups=None, n_jobs=-1, use_gpu=False, custom_pipeline=None, html=True, session_id=786, log_experiment=False, experiment_name=None, log_plots=False, log_profile=False, log_data=False, silent=False, verbose=True, profile=False, profile_kwargs=None, display=None)
2021-02-16 10:29:33,550:INFO:Checking environment
2021-02-16 10:29:33,550:INFO:python_version: 3.7.4
2021-02-16 10:29:33,550:INFO:python_build: ('default', 'Aug  9 2019 18:34:13')
2021-02-16 10:29:33,550:INFO:machine: AMD64
2021-02-16 10:29:33,550:INFO:platform: Windows-10-10.0.19041-SP0
2021-02-16 10:29:33,551:INFO:Memory: svmem(total=8504934400, available=1337614336, percent=84.3, used=7167320064, free=1337614336)
2021-02-16 10:29:33,551:INFO:Physical Core: 2
2021-02-16 10:29:33,551:INFO:Logical Core: 4
2021-02-16 10:29:33,551:INFO:Checking libraries
2021-02-16 10:29:33,551:INFO:pd==0.25.3
2021-02-16 10:29:33,551:INFO:numpy==1.18.1
2021-02-16 10:29:33,551:INFO:sklearn==0.22.1
2021-02-16 10:29:33,551:INFO:xgboost==0.90
2021-02-16 10:29:33,551:INFO:lightgbm==3.1.1
2021-02-16 10:29:33,551:INFO:catboost==0.24.4
2021-02-16 10:29:34,885:INFO:mlflow==1.13.1
2021-02-16 10:29:34,885:INFO:Checking Exceptions
2021-02-16 10:29:34,886:INFO:Declaring global variables
2021-02-16 10:29:34,886:INFO:USI: 3640
2021-02-16 10:29:34,886:INFO:pycaret_globals: {'_gpu_n_jobs_param', 'display_container', 'transform_target_method_param', 'target_param', '_all_models', 'transform_target_param', 'gpu_param', 'data_before_preprocess', 'master_model_container', 'html_param', 'X_train', 'experiment__', 'X', 'iterative_imputation_iters_param', 'create_model_container', 'prep_pipe', 'exp_name_log', 'pycaret_globals', 'imputation_classifier', 'fold_generator', '_ml_usecase', 'y_test', 'seed', 'log_plots_param', 'imputation_regressor', '_internal_pipeline', 'fold_groups_param', 'n_jobs_param', 'logging_param', '_all_models_internal', 'stratify_param', 'X_test', '_available_plots', 'fold_param', 'fold_shuffle_param', 'fix_imbalance_method_param', 'USI', 'y', '_all_metrics', 'fix_imbalance_param', 'y_train'}
2021-02-16 10:29:34,886:INFO:Preparing display monitor
2021-02-16 10:29:34,886:INFO:Preparing display monitor
2021-02-16 10:29:34,907:INFO:Importing libraries
2021-02-16 10:29:34,907:INFO:Copying data for preprocessing
2021-02-16 10:29:34,920:INFO:Declaring preprocessing parameters
2021-02-16 10:29:34,926:INFO:Creating preprocessing pipeline
2021-02-16 10:29:34,938:INFO:Preprocessing pipeline created successfully
2021-02-16 10:29:34,939:ERROR:(Process Exit): setup has been interupted with user command 'quit'. setup must rerun.
2021-02-16 10:29:34,939:INFO:Creating global containers
2021-02-16 10:29:34,939:INFO:Internal pipeline: Pipeline(memory=None, steps=[('empty_step', 'passthrough')], verbose=False)
2021-02-16 10:29:36,124:INFO:Creating grid variables
2021-02-16 10:29:36,211:INFO:create_model_container: 0
2021-02-16 10:29:36,212:INFO:master_model_container: 0
2021-02-16 10:29:36,212:INFO:display_container: 1
2021-02-16 10:29:36,220:INFO:Pipeline(memory=None,
         steps=[('dtypes',
                 DataTypes_Auto_infer(categorical_features=[],
                                      display_types=True, features_todrop=[],
                                      id_columns=[], ml_usecase='regression',
                                      numerical_features=['day', 'hour',
                                                          'humidity', 'month',
                                                          'temp', 'weekend',
                                                          'windspeed_log',
                                                          'workingday', 'year',
                                                          'season_2',
                                                          'season_3',
                                                          'season_4',
                                                          'weather_2',
                                                          'weather_3',
                                                          'weather_4'],
                                      target='count_lo...
                ('scaling', 'passthrough'), ('P_transform', 'passthrough'),
                ('binn', 'passthrough'), ('rem_outliers', 'passthrough'),
                ('cluster_all', 'passthrough'),
                ('dummy', Dummify(target='count_log')),
                ('fix_perfect', Remove_100(target='count_log')),
                ('clean_names', Clean_Colum_Names()),
                ('feature_select', 'passthrough'), ('fix_multi', 'passthrough'),
                ('dfs', 'passthrough'), ('pca', 'passthrough')],
         verbose=False)
2021-02-16 10:29:36,221:INFO:setup() succesfully completed......................................
2021-02-16 10:29:37,422:INFO:Initializing compare_models()
2021-02-16 10:29:37,422:INFO:compare_models(include=None, fold=None, round=4, cross_validation=True, sort=R2, n_select=1, budget_time=None, turbo=True, errors=ignore, fit_kwargs=None, groups=None, verbose=True, display=None, exclude=None)
2021-02-16 10:29:37,422:INFO:Checking exceptions
2021-02-16 10:29:37,423:INFO:Preparing display monitor
2021-02-16 10:29:37,423:INFO:Preparing display monitor
2021-02-16 10:29:37,494:INFO:Initializing Linear Regression
2021-02-16 10:29:37,494:INFO:Total runtime is 1.6673405965169272e-05 minutes
2021-02-16 10:29:37,510:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:37,511:INFO:Initializing create_model()
2021-02-16 10:29:37,511:INFO:create_model(estimator=lr, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:37,511:INFO:Checking exceptions
2021-02-16 10:29:37,511:INFO:Importing libraries
2021-02-16 10:29:37,512:INFO:Copying training dataset
2021-02-16 10:29:37,515:INFO:Defining folds
2021-02-16 10:29:37,515:INFO:Declaring metric variables
2021-02-16 10:29:37,532:INFO:Importing untrained model
2021-02-16 10:29:37,551:INFO:Linear Regression Imported succesfully
2021-02-16 10:29:37,593:INFO:Starting cross validation
2021-02-16 10:29:37,606:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:45,840:INFO:Calculating mean and std
2021-02-16 10:29:45,841:INFO:Creating metrics dataframe
2021-02-16 10:29:45,852:INFO:Uploading results into container
2021-02-16 10:29:45,853:INFO:Uploading model into container now
2021-02-16 10:29:45,853:INFO:create_model_container: 1
2021-02-16 10:29:45,853:INFO:master_model_container: 1
2021-02-16 10:29:45,854:INFO:display_container: 2
2021-02-16 10:29:45,854:INFO:LinearRegression(copy_X=True, fit_intercept=True, n_jobs=-1, normalize=False)
2021-02-16 10:29:45,854:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:46,924:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:46,924:INFO:Creating metrics dataframe
2021-02-16 10:29:46,942:INFO:Initializing Lasso Regression
2021-02-16 10:29:46,943:INFO:Total runtime is 0.1574895699818929 minutes
2021-02-16 10:29:46,957:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:46,958:INFO:Initializing create_model()
2021-02-16 10:29:46,958:INFO:create_model(estimator=lasso, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:46,958:INFO:Checking exceptions
2021-02-16 10:29:46,958:INFO:Importing libraries
2021-02-16 10:29:46,958:INFO:Copying training dataset
2021-02-16 10:29:46,960:INFO:Defining folds
2021-02-16 10:29:46,961:INFO:Declaring metric variables
2021-02-16 10:29:46,973:INFO:Importing untrained model
2021-02-16 10:29:46,998:INFO:Lasso Regression Imported succesfully
2021-02-16 10:29:47,029:INFO:Starting cross validation
2021-02-16 10:29:47,031:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:47,201:INFO:Calculating mean and std
2021-02-16 10:29:47,203:INFO:Creating metrics dataframe
2021-02-16 10:29:47,214:INFO:Uploading results into container
2021-02-16 10:29:47,214:INFO:Uploading model into container now
2021-02-16 10:29:47,215:INFO:create_model_container: 2
2021-02-16 10:29:47,215:INFO:master_model_container: 2
2021-02-16 10:29:47,215:INFO:display_container: 2
2021-02-16 10:29:47,216:INFO:Lasso(alpha=1.0, copy_X=True, fit_intercept=True, max_iter=1000,
      normalize=False, positive=False, precompute=False, random_state=786,
      selection='cyclic', tol=0.0001, warm_start=False)
2021-02-16 10:29:47,216:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:48,281:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:48,281:INFO:Creating metrics dataframe
2021-02-16 10:29:48,301:INFO:Initializing Ridge Regression
2021-02-16 10:29:48,301:INFO:Total runtime is 0.18012654383977253 minutes
2021-02-16 10:29:48,317:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:48,318:INFO:Initializing create_model()
2021-02-16 10:29:48,318:INFO:create_model(estimator=ridge, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:48,318:INFO:Checking exceptions
2021-02-16 10:29:48,318:INFO:Importing libraries
2021-02-16 10:29:48,318:INFO:Copying training dataset
2021-02-16 10:29:48,321:INFO:Defining folds
2021-02-16 10:29:48,321:INFO:Declaring metric variables
2021-02-16 10:29:48,339:INFO:Importing untrained model
2021-02-16 10:29:48,361:INFO:Ridge Regression Imported succesfully
2021-02-16 10:29:48,401:INFO:Starting cross validation
2021-02-16 10:29:48,402:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:51,508:INFO:Calculating mean and std
2021-02-16 10:29:51,509:INFO:Creating metrics dataframe
2021-02-16 10:29:51,515:INFO:Uploading results into container
2021-02-16 10:29:51,515:INFO:Uploading model into container now
2021-02-16 10:29:51,516:INFO:create_model_container: 3
2021-02-16 10:29:51,516:INFO:master_model_container: 3
2021-02-16 10:29:51,516:INFO:display_container: 2
2021-02-16 10:29:51,516:INFO:Ridge(alpha=1.0, copy_X=True, fit_intercept=True, max_iter=None,
      normalize=False, random_state=786, solver='auto', tol=0.001)
2021-02-16 10:29:51,516:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:52,482:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:52,482:INFO:Creating metrics dataframe
2021-02-16 10:29:52,508:INFO:Initializing Elastic Net
2021-02-16 10:29:52,508:INFO:Total runtime is 0.25023971001307166 minutes
2021-02-16 10:29:52,523:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:52,524:INFO:Initializing create_model()
2021-02-16 10:29:52,524:INFO:create_model(estimator=en, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:52,525:INFO:Checking exceptions
2021-02-16 10:29:52,525:INFO:Importing libraries
2021-02-16 10:29:52,525:INFO:Copying training dataset
2021-02-16 10:29:52,526:INFO:Defining folds
2021-02-16 10:29:52,526:INFO:Declaring metric variables
2021-02-16 10:29:52,540:INFO:Importing untrained model
2021-02-16 10:29:52,562:INFO:Elastic Net Imported succesfully
2021-02-16 10:29:52,600:INFO:Starting cross validation
2021-02-16 10:29:52,601:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:52,810:INFO:Calculating mean and std
2021-02-16 10:29:52,812:INFO:Creating metrics dataframe
2021-02-16 10:29:52,820:INFO:Uploading results into container
2021-02-16 10:29:52,820:INFO:Uploading model into container now
2021-02-16 10:29:52,821:INFO:create_model_container: 4
2021-02-16 10:29:52,821:INFO:master_model_container: 4
2021-02-16 10:29:52,821:INFO:display_container: 2
2021-02-16 10:29:52,822:INFO:ElasticNet(alpha=1.0, copy_X=True, fit_intercept=True, l1_ratio=0.5,
           max_iter=1000, normalize=False, positive=False, precompute=False,
           random_state=786, selection='cyclic', tol=0.0001, warm_start=False)
2021-02-16 10:29:52,822:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:53,824:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:53,824:INFO:Creating metrics dataframe
2021-02-16 10:29:53,859:INFO:Initializing Least Angle Regression
2021-02-16 10:29:53,860:INFO:Total runtime is 0.2727752844492594 minutes
2021-02-16 10:29:53,885:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:53,886:INFO:Initializing create_model()
2021-02-16 10:29:53,886:INFO:create_model(estimator=lar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:53,886:INFO:Checking exceptions
2021-02-16 10:29:53,886:INFO:Importing libraries
2021-02-16 10:29:53,886:INFO:Copying training dataset
2021-02-16 10:29:53,890:INFO:Defining folds
2021-02-16 10:29:53,890:INFO:Declaring metric variables
2021-02-16 10:29:53,918:INFO:Importing untrained model
2021-02-16 10:29:53,918:WARNING:create_model() for lar raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 10:29:54,627:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 10:29:54,627:INFO:Initializing create_model()
2021-02-16 10:29:54,627:INFO:create_model(estimator=lar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:54,628:INFO:Checking exceptions
2021-02-16 10:29:54,628:INFO:Importing libraries
2021-02-16 10:29:54,628:INFO:Copying training dataset
2021-02-16 10:29:54,633:INFO:Defining folds
2021-02-16 10:29:54,634:INFO:Declaring metric variables
2021-02-16 10:29:54,659:INFO:Importing untrained model
2021-02-16 10:29:54,659:ERROR:create_model() for lar raised an exception:
2021-02-16 10:29:54,660:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 10:29:54,660:INFO:Initializing Lasso Least Angle Regression
2021-02-16 10:29:54,660:INFO:Total runtime is 0.2861103336016337 minutes
2021-02-16 10:29:54,672:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:54,672:INFO:Initializing create_model()
2021-02-16 10:29:54,673:INFO:create_model(estimator=llar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:54,673:INFO:Checking exceptions
2021-02-16 10:29:54,673:INFO:Importing libraries
2021-02-16 10:29:54,673:INFO:Copying training dataset
2021-02-16 10:29:54,675:INFO:Defining folds
2021-02-16 10:29:54,675:INFO:Declaring metric variables
2021-02-16 10:29:54,692:INFO:Importing untrained model
2021-02-16 10:29:54,693:WARNING:create_model() for llar raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 10:29:54,693:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 10:29:54,694:INFO:Initializing create_model()
2021-02-16 10:29:54,694:INFO:create_model(estimator=llar, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:54,694:INFO:Checking exceptions
2021-02-16 10:29:54,694:INFO:Importing libraries
2021-02-16 10:29:54,695:INFO:Copying training dataset
2021-02-16 10:29:54,697:INFO:Defining folds
2021-02-16 10:29:54,698:INFO:Declaring metric variables
2021-02-16 10:29:54,708:INFO:Importing untrained model
2021-02-16 10:29:54,709:ERROR:create_model() for llar raised an exception:
2021-02-16 10:29:54,709:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2979, in create_model_supervised
    model = model_definition.class_def(**model_args)
TypeError: __init__() got an unexpected keyword argument 'random_state'

2021-02-16 10:29:54,709:INFO:Initializing Orthogonal Matching Pursuit
2021-02-16 10:29:54,709:INFO:Total runtime is 0.28692786296208694 minutes
2021-02-16 10:29:54,722:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:54,722:INFO:Initializing create_model()
2021-02-16 10:29:54,722:INFO:create_model(estimator=omp, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:54,722:INFO:Checking exceptions
2021-02-16 10:29:54,722:INFO:Importing libraries
2021-02-16 10:29:54,722:INFO:Copying training dataset
2021-02-16 10:29:54,724:INFO:Defining folds
2021-02-16 10:29:54,724:INFO:Declaring metric variables
2021-02-16 10:29:54,735:INFO:Importing untrained model
2021-02-16 10:29:54,748:INFO:Orthogonal Matching Pursuit Imported succesfully
2021-02-16 10:29:54,798:INFO:Starting cross validation
2021-02-16 10:29:54,800:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:54,949:INFO:Calculating mean and std
2021-02-16 10:29:54,950:INFO:Creating metrics dataframe
2021-02-16 10:29:54,956:INFO:Uploading results into container
2021-02-16 10:29:54,956:INFO:Uploading model into container now
2021-02-16 10:29:54,956:INFO:create_model_container: 5
2021-02-16 10:29:54,956:INFO:master_model_container: 5
2021-02-16 10:29:54,956:INFO:display_container: 2
2021-02-16 10:29:54,957:INFO:OrthogonalMatchingPursuit(fit_intercept=True, n_nonzero_coefs=None,
                          normalize=True, precompute='auto', tol=None)
2021-02-16 10:29:54,957:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:55,972:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:55,972:INFO:Creating metrics dataframe
2021-02-16 10:29:56,076:INFO:Initializing Bayesian Ridge
2021-02-16 10:29:56,077:INFO:Total runtime is 0.30972967545191443 minutes
2021-02-16 10:29:56,119:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:56,120:INFO:Initializing create_model()
2021-02-16 10:29:56,121:INFO:create_model(estimator=br, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:56,121:INFO:Checking exceptions
2021-02-16 10:29:56,121:INFO:Importing libraries
2021-02-16 10:29:56,121:INFO:Copying training dataset
2021-02-16 10:29:56,123:INFO:Defining folds
2021-02-16 10:29:56,123:INFO:Declaring metric variables
2021-02-16 10:29:56,140:INFO:Importing untrained model
2021-02-16 10:29:56,180:INFO:Bayesian Ridge Imported succesfully
2021-02-16 10:29:56,239:INFO:Starting cross validation
2021-02-16 10:29:56,240:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:56,808:INFO:Calculating mean and std
2021-02-16 10:29:56,811:INFO:Creating metrics dataframe
2021-02-16 10:29:56,825:INFO:Uploading results into container
2021-02-16 10:29:56,825:INFO:Uploading model into container now
2021-02-16 10:29:56,825:INFO:create_model_container: 6
2021-02-16 10:29:56,825:INFO:master_model_container: 6
2021-02-16 10:29:56,826:INFO:display_container: 2
2021-02-16 10:29:56,826:INFO:BayesianRidge(alpha_1=1e-06, alpha_2=1e-06, alpha_init=None,
              compute_score=False, copy_X=True, fit_intercept=True,
              lambda_1=1e-06, lambda_2=1e-06, lambda_init=None, n_iter=300,
              normalize=False, tol=0.001, verbose=False)
2021-02-16 10:29:56,827:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:57,884:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:57,884:INFO:Creating metrics dataframe
2021-02-16 10:29:57,915:INFO:Initializing Passive Aggressive Regressor
2021-02-16 10:29:57,916:INFO:Total runtime is 0.34037367105484007 minutes
2021-02-16 10:29:57,930:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:57,931:INFO:Initializing create_model()
2021-02-16 10:29:57,931:INFO:create_model(estimator=par, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:57,931:INFO:Checking exceptions
2021-02-16 10:29:57,931:INFO:Importing libraries
2021-02-16 10:29:57,931:INFO:Copying training dataset
2021-02-16 10:29:57,933:INFO:Defining folds
2021-02-16 10:29:57,933:INFO:Declaring metric variables
2021-02-16 10:29:57,945:INFO:Importing untrained model
2021-02-16 10:29:57,965:INFO:Passive Aggressive Regressor Imported succesfully
2021-02-16 10:29:57,998:INFO:Starting cross validation
2021-02-16 10:29:58,000:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:29:58,237:INFO:Calculating mean and std
2021-02-16 10:29:58,238:INFO:Creating metrics dataframe
2021-02-16 10:29:58,244:INFO:Uploading results into container
2021-02-16 10:29:58,244:INFO:Uploading model into container now
2021-02-16 10:29:58,244:INFO:create_model_container: 7
2021-02-16 10:29:58,244:INFO:master_model_container: 7
2021-02-16 10:29:58,245:INFO:display_container: 2
2021-02-16 10:29:58,245:INFO:PassiveAggressiveRegressor(C=1.0, average=False, early_stopping=False,
                           epsilon=0.1, fit_intercept=True,
                           loss='epsilon_insensitive', max_iter=1000,
                           n_iter_no_change=5, random_state=786, shuffle=True,
                           tol=0.001, validation_fraction=0.1, verbose=0,
                           warm_start=False)
2021-02-16 10:29:58,245:INFO:create_model() succesfully completed......................................
2021-02-16 10:29:59,151:INFO:SubProcess create_model() end ==================================
2021-02-16 10:29:59,151:INFO:Creating metrics dataframe
2021-02-16 10:29:59,182:INFO:Initializing Huber Regressor
2021-02-16 10:29:59,182:INFO:Total runtime is 0.3614782333374023 minutes
2021-02-16 10:29:59,196:INFO:SubProcess create_model() called ==================================
2021-02-16 10:29:59,196:INFO:Initializing create_model()
2021-02-16 10:29:59,197:INFO:create_model(estimator=huber, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:29:59,197:INFO:Checking exceptions
2021-02-16 10:29:59,197:INFO:Importing libraries
2021-02-16 10:29:59,197:INFO:Copying training dataset
2021-02-16 10:29:59,199:INFO:Defining folds
2021-02-16 10:29:59,199:INFO:Declaring metric variables
2021-02-16 10:29:59,211:INFO:Importing untrained model
2021-02-16 10:29:59,232:INFO:Huber Regressor Imported succesfully
2021-02-16 10:29:59,265:INFO:Starting cross validation
2021-02-16 10:29:59,266:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:00,669:INFO:Calculating mean and std
2021-02-16 10:30:00,671:INFO:Creating metrics dataframe
2021-02-16 10:30:00,681:INFO:Uploading results into container
2021-02-16 10:30:00,681:INFO:Uploading model into container now
2021-02-16 10:30:00,681:INFO:create_model_container: 8
2021-02-16 10:30:00,681:INFO:master_model_container: 8
2021-02-16 10:30:00,681:INFO:display_container: 2
2021-02-16 10:30:00,682:INFO:HuberRegressor(alpha=0.0001, epsilon=1.35, fit_intercept=True, max_iter=100,
               tol=1e-05, warm_start=False)
2021-02-16 10:30:00,682:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:01,597:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:01,597:INFO:Creating metrics dataframe
2021-02-16 10:30:01,633:INFO:Initializing K Neighbors Regressor
2021-02-16 10:30:01,633:INFO:Total runtime is 0.4023285150527954 minutes
2021-02-16 10:30:01,648:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:01,649:INFO:Initializing create_model()
2021-02-16 10:30:01,649:INFO:create_model(estimator=knn, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:01,649:INFO:Checking exceptions
2021-02-16 10:30:01,649:INFO:Importing libraries
2021-02-16 10:30:01,649:INFO:Copying training dataset
2021-02-16 10:30:01,650:INFO:Defining folds
2021-02-16 10:30:01,651:INFO:Declaring metric variables
2021-02-16 10:30:01,666:INFO:Importing untrained model
2021-02-16 10:30:01,687:INFO:K Neighbors Regressor Imported succesfully
2021-02-16 10:30:01,721:INFO:Starting cross validation
2021-02-16 10:30:01,723:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:02,702:INFO:Calculating mean and std
2021-02-16 10:30:02,706:INFO:Creating metrics dataframe
2021-02-16 10:30:02,722:INFO:Uploading results into container
2021-02-16 10:30:02,722:INFO:Uploading model into container now
2021-02-16 10:30:02,723:INFO:create_model_container: 9
2021-02-16 10:30:02,723:INFO:master_model_container: 9
2021-02-16 10:30:02,724:INFO:display_container: 2
2021-02-16 10:30:02,725:INFO:KNeighborsRegressor(algorithm='auto', leaf_size=30, metric='minkowski',
                    metric_params=None, n_jobs=-1, n_neighbors=5, p=2,
                    weights='uniform')
2021-02-16 10:30:02,725:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:03,667:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:03,668:INFO:Creating metrics dataframe
2021-02-16 10:30:03,706:INFO:Initializing Decision Tree Regressor
2021-02-16 10:30:03,707:INFO:Total runtime is 0.4368931174278259 minutes
2021-02-16 10:30:03,722:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:03,722:INFO:Initializing create_model()
2021-02-16 10:30:03,723:INFO:create_model(estimator=dt, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:03,723:INFO:Checking exceptions
2021-02-16 10:30:03,723:INFO:Importing libraries
2021-02-16 10:30:03,723:INFO:Copying training dataset
2021-02-16 10:30:03,725:INFO:Defining folds
2021-02-16 10:30:03,725:INFO:Declaring metric variables
2021-02-16 10:30:03,740:INFO:Importing untrained model
2021-02-16 10:30:03,764:INFO:Decision Tree Regressor Imported succesfully
2021-02-16 10:30:03,800:INFO:Starting cross validation
2021-02-16 10:30:03,801:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:04,106:INFO:Calculating mean and std
2021-02-16 10:30:04,108:INFO:Creating metrics dataframe
2021-02-16 10:30:04,114:INFO:Uploading results into container
2021-02-16 10:30:04,114:INFO:Uploading model into container now
2021-02-16 10:30:04,114:INFO:create_model_container: 10
2021-02-16 10:30:04,115:INFO:master_model_container: 10
2021-02-16 10:30:04,115:INFO:display_container: 2
2021-02-16 10:30:04,115:INFO:DecisionTreeRegressor(ccp_alpha=0.0, criterion='mse', max_depth=None,
                      max_features=None, max_leaf_nodes=None,
                      min_impurity_decrease=0.0, min_impurity_split=None,
                      min_samples_leaf=1, min_samples_split=2,
                      min_weight_fraction_leaf=0.0, presort='deprecated',
                      random_state=786, splitter='best')
2021-02-16 10:30:04,115:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:05,160:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:05,160:INFO:Creating metrics dataframe
2021-02-16 10:30:05,215:INFO:Initializing Random Forest Regressor
2021-02-16 10:30:05,215:INFO:Total runtime is 0.46202870607376095 minutes
2021-02-16 10:30:05,232:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:05,232:INFO:Initializing create_model()
2021-02-16 10:30:05,233:INFO:create_model(estimator=rf, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:05,233:INFO:Checking exceptions
2021-02-16 10:30:05,233:INFO:Importing libraries
2021-02-16 10:30:05,233:INFO:Copying training dataset
2021-02-16 10:30:05,236:INFO:Defining folds
2021-02-16 10:30:05,236:INFO:Declaring metric variables
2021-02-16 10:30:05,251:INFO:Importing untrained model
2021-02-16 10:30:05,270:INFO:Random Forest Regressor Imported succesfully
2021-02-16 10:30:05,306:INFO:Starting cross validation
2021-02-16 10:30:05,307:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:20,239:INFO:Calculating mean and std
2021-02-16 10:30:20,243:INFO:Creating metrics dataframe
2021-02-16 10:30:20,263:INFO:Uploading results into container
2021-02-16 10:30:20,264:INFO:Uploading model into container now
2021-02-16 10:30:20,265:INFO:create_model_container: 11
2021-02-16 10:30:20,265:INFO:master_model_container: 11
2021-02-16 10:30:20,265:INFO:display_container: 2
2021-02-16 10:30:20,268:INFO:RandomForestRegressor(bootstrap=True, ccp_alpha=0.0, criterion='mse',
                      max_depth=None, max_features='auto', max_leaf_nodes=None,
                      max_samples=None, min_impurity_decrease=0.0,
                      min_impurity_split=None, min_samples_leaf=1,
                      min_samples_split=2, min_weight_fraction_leaf=0.0,
                      n_estimators=100, n_jobs=-1, oob_score=False,
                      random_state=786, verbose=0, warm_start=False)
2021-02-16 10:30:20,268:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:21,196:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:21,196:INFO:Creating metrics dataframe
2021-02-16 10:30:21,249:INFO:Initializing Extra Trees Regressor
2021-02-16 10:30:21,250:INFO:Total runtime is 0.7292794903119405 minutes
2021-02-16 10:30:21,267:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:21,268:INFO:Initializing create_model()
2021-02-16 10:30:21,269:INFO:create_model(estimator=et, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:21,269:INFO:Checking exceptions
2021-02-16 10:30:21,269:INFO:Importing libraries
2021-02-16 10:30:21,269:INFO:Copying training dataset
2021-02-16 10:30:21,271:INFO:Defining folds
2021-02-16 10:30:21,271:INFO:Declaring metric variables
2021-02-16 10:30:21,290:INFO:Importing untrained model
2021-02-16 10:30:21,319:INFO:Extra Trees Regressor Imported succesfully
2021-02-16 10:30:21,364:INFO:Starting cross validation
2021-02-16 10:30:21,365:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:32,766:INFO:Calculating mean and std
2021-02-16 10:30:32,770:INFO:Creating metrics dataframe
2021-02-16 10:30:32,787:INFO:Uploading results into container
2021-02-16 10:30:32,787:INFO:Uploading model into container now
2021-02-16 10:30:32,788:INFO:create_model_container: 12
2021-02-16 10:30:32,788:INFO:master_model_container: 12
2021-02-16 10:30:32,789:INFO:display_container: 2
2021-02-16 10:30:32,790:INFO:ExtraTreesRegressor(bootstrap=False, ccp_alpha=0.0, criterion='mse',
                    max_depth=None, max_features='auto', max_leaf_nodes=None,
                    max_samples=None, min_impurity_decrease=0.0,
                    min_impurity_split=None, min_samples_leaf=1,
                    min_samples_split=2, min_weight_fraction_leaf=0.0,
                    n_estimators=100, n_jobs=-1, oob_score=False,
                    random_state=786, verbose=0, warm_start=False)
2021-02-16 10:30:32,791:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:33,715:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:33,715:INFO:Creating metrics dataframe
2021-02-16 10:30:33,758:INFO:Initializing AdaBoost Regressor
2021-02-16 10:30:33,758:INFO:Total runtime is 0.9377451221148173 minutes
2021-02-16 10:30:33,771:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:33,772:INFO:Initializing create_model()
2021-02-16 10:30:33,772:INFO:create_model(estimator=ada, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:33,772:INFO:Checking exceptions
2021-02-16 10:30:33,772:INFO:Importing libraries
2021-02-16 10:30:33,773:INFO:Copying training dataset
2021-02-16 10:30:33,774:INFO:Defining folds
2021-02-16 10:30:33,774:INFO:Declaring metric variables
2021-02-16 10:30:33,787:INFO:Importing untrained model
2021-02-16 10:30:33,810:INFO:AdaBoost Regressor Imported succesfully
2021-02-16 10:30:33,853:INFO:Starting cross validation
2021-02-16 10:30:33,854:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:36,211:INFO:Calculating mean and std
2021-02-16 10:30:36,212:INFO:Creating metrics dataframe
2021-02-16 10:30:36,218:INFO:Uploading results into container
2021-02-16 10:30:36,218:INFO:Uploading model into container now
2021-02-16 10:30:36,218:INFO:create_model_container: 13
2021-02-16 10:30:36,218:INFO:master_model_container: 13
2021-02-16 10:30:36,218:INFO:display_container: 2
2021-02-16 10:30:36,219:INFO:AdaBoostRegressor(base_estimator=None, learning_rate=1.0, loss='linear',
                  n_estimators=50, random_state=786)
2021-02-16 10:30:36,219:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:37,114:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:37,114:INFO:Creating metrics dataframe
2021-02-16 10:30:37,161:INFO:Initializing Gradient Boosting Regressor
2021-02-16 10:30:37,161:INFO:Total runtime is 0.9944681922594706 minutes
2021-02-16 10:30:37,175:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:37,176:INFO:Initializing create_model()
2021-02-16 10:30:37,176:INFO:create_model(estimator=gbr, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:37,176:INFO:Checking exceptions
2021-02-16 10:30:37,176:INFO:Importing libraries
2021-02-16 10:30:37,177:INFO:Copying training dataset
2021-02-16 10:30:37,178:INFO:Defining folds
2021-02-16 10:30:37,179:INFO:Declaring metric variables
2021-02-16 10:30:37,193:INFO:Importing untrained model
2021-02-16 10:30:37,217:INFO:Gradient Boosting Regressor Imported succesfully
2021-02-16 10:30:37,260:INFO:Starting cross validation
2021-02-16 10:30:37,261:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:40,214:INFO:Calculating mean and std
2021-02-16 10:30:40,215:INFO:Creating metrics dataframe
2021-02-16 10:30:40,221:INFO:Uploading results into container
2021-02-16 10:30:40,221:INFO:Uploading model into container now
2021-02-16 10:30:40,221:INFO:create_model_container: 14
2021-02-16 10:30:40,222:INFO:master_model_container: 14
2021-02-16 10:30:40,222:INFO:display_container: 2
2021-02-16 10:30:40,222:INFO:GradientBoostingRegressor(alpha=0.9, ccp_alpha=0.0, criterion='friedman_mse',
                          init=None, learning_rate=0.1, loss='ls', max_depth=3,
                          max_features=None, max_leaf_nodes=None,
                          min_impurity_decrease=0.0, min_impurity_split=None,
                          min_samples_leaf=1, min_samples_split=2,
                          min_weight_fraction_leaf=0.0, n_estimators=100,
                          n_iter_no_change=None, presort='deprecated',
                          random_state=786, subsample=1.0, tol=0.0001,
                          validation_fraction=0.1, verbose=0, warm_start=False)
2021-02-16 10:30:40,223:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:41,136:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:41,137:INFO:Creating metrics dataframe
2021-02-16 10:30:41,185:INFO:Initializing Extreme Gradient Boosting
2021-02-16 10:30:41,185:INFO:Total runtime is 1.0615298430124918 minutes
2021-02-16 10:30:41,199:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:41,200:INFO:Initializing create_model()
2021-02-16 10:30:41,200:INFO:create_model(estimator=xgboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:41,200:INFO:Checking exceptions
2021-02-16 10:30:41,200:INFO:Importing libraries
2021-02-16 10:30:41,200:INFO:Copying training dataset
2021-02-16 10:30:41,202:INFO:Defining folds
2021-02-16 10:30:41,202:INFO:Declaring metric variables
2021-02-16 10:30:41,214:INFO:Importing untrained model
2021-02-16 10:30:41,233:INFO:Extreme Gradient Boosting Imported succesfully
2021-02-16 10:30:41,274:INFO:Starting cross validation
2021-02-16 10:30:41,275:WARNING:create_model() for xgboost raised an exception or returned all 0.0, trying without fit_kwargs:
2021-02-16 10:30:41,312:WARNING:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

2021-02-16 10:30:41,312:INFO:Initializing create_model()
2021-02-16 10:30:41,312:INFO:create_model(estimator=xgboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:41,312:INFO:Checking exceptions
2021-02-16 10:30:41,312:INFO:Importing libraries
2021-02-16 10:30:41,312:INFO:Copying training dataset
2021-02-16 10:30:41,314:INFO:Defining folds
2021-02-16 10:30:41,314:INFO:Declaring metric variables
2021-02-16 10:30:41,324:INFO:Importing untrained model
2021-02-16 10:30:41,338:INFO:Extreme Gradient Boosting Imported succesfully
2021-02-16 10:30:41,367:INFO:Starting cross validation
2021-02-16 10:30:41,368:ERROR:create_model() for xgboost raised an exception:
2021-02-16 10:30:41,369:ERROR:Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2151, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 217, in add_estimator_to_pipeline
    assert hasattr(pipeline._final_estimator, "predict")
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 2170, in compare_models
    refit=False,
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\tabular.py", line 3085, in create_model_supervised
    with estimator_pipeline(_internal_pipeline, model) as pipeline_with_model:
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 199, in __enter__
    add_estimator_to_pipeline(self.pipeline, self.estimator)
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 222, in add_estimator_to_pipeline
    pipeline._carry_over_final_estimator_fit_vars()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\pipeline.py", line 58, in _carry_over_final_estimator_fit_vars
    self._final_estimator
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 571, in get_all_object_vars_and_properties
    for k in object.__dir__()
  File "C:\Users\affine\AppData\Roaming\Python\Python37\site-packages\pycaret\internal\utils.py", line 572, in <dictcomp>
    if k[:2] != "__" and type(getattr(object, k, "")).__name__ != "method"
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 542, in feature_importances_
    b = self.get_booster()
  File "D:\Anaconda3\lib\site-packages\xgboost\sklearn.py", line 193, in get_booster
    raise XGBoostError('need to call fit or load_model beforehand')
xgboost.core.XGBoostError: need to call fit or load_model beforehand

2021-02-16 10:30:41,370:INFO:Initializing Light Gradient Boosting Machine
2021-02-16 10:30:41,370:INFO:Total runtime is 1.0646130323410032 minutes
2021-02-16 10:30:41,385:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:41,386:INFO:Initializing create_model()
2021-02-16 10:30:41,386:INFO:create_model(estimator=lightgbm, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:41,386:INFO:Checking exceptions
2021-02-16 10:30:41,387:INFO:Importing libraries
2021-02-16 10:30:41,387:INFO:Copying training dataset
2021-02-16 10:30:41,389:INFO:Defining folds
2021-02-16 10:30:41,389:INFO:Declaring metric variables
2021-02-16 10:30:41,403:INFO:Importing untrained model
2021-02-16 10:30:41,418:INFO:Light Gradient Boosting Machine Imported succesfully
2021-02-16 10:30:41,450:INFO:Starting cross validation
2021-02-16 10:30:41,451:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:30:42,266:INFO:Calculating mean and std
2021-02-16 10:30:42,267:INFO:Creating metrics dataframe
2021-02-16 10:30:42,274:INFO:Uploading results into container
2021-02-16 10:30:42,274:INFO:Uploading model into container now
2021-02-16 10:30:42,274:INFO:create_model_container: 15
2021-02-16 10:30:42,274:INFO:master_model_container: 15
2021-02-16 10:30:42,274:INFO:display_container: 2
2021-02-16 10:30:42,275:INFO:LGBMRegressor(boosting_type='gbdt', class_weight=None, colsample_bytree=1.0,
              importance_type='split', learning_rate=0.1, max_depth=-1,
              min_child_samples=20, min_child_weight=0.001, min_split_gain=0.0,
              n_estimators=100, n_jobs=-1, num_leaves=31, objective=None,
              random_state=786, reg_alpha=0.0, reg_lambda=0.0, silent=True,
              subsample=1.0, subsample_for_bin=200000, subsample_freq=0)
2021-02-16 10:30:42,276:INFO:create_model() succesfully completed......................................
2021-02-16 10:30:43,187:INFO:SubProcess create_model() end ==================================
2021-02-16 10:30:43,187:INFO:Creating metrics dataframe
2021-02-16 10:30:43,237:INFO:Initializing CatBoost Regressor
2021-02-16 10:30:43,237:INFO:Total runtime is 1.0957312663396197 minutes
2021-02-16 10:30:43,251:INFO:SubProcess create_model() called ==================================
2021-02-16 10:30:43,252:INFO:Initializing create_model()
2021-02-16 10:30:43,252:INFO:create_model(estimator=catboost, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=True, predict=True, fit_kwargs={}, groups=None, refit=False, verbose=False, system=False, metrics=None, display=<pycaret.internal.Display.Display object at 0x00000236F1F2CE08>, kwargs={})
2021-02-16 10:30:43,252:INFO:Checking exceptions
2021-02-16 10:30:43,252:INFO:Importing libraries
2021-02-16 10:30:43,252:INFO:Copying training dataset
2021-02-16 10:30:43,254:INFO:Defining folds
2021-02-16 10:30:43,255:INFO:Declaring metric variables
2021-02-16 10:30:43,270:INFO:Importing untrained model
2021-02-16 10:30:43,313:INFO:CatBoost Regressor Imported succesfully
2021-02-16 10:30:43,366:INFO:Starting cross validation
2021-02-16 10:30:43,367:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:31:36,727:INFO:Calculating mean and std
2021-02-16 10:31:36,730:INFO:Creating metrics dataframe
2021-02-16 10:31:36,740:INFO:Uploading results into container
2021-02-16 10:31:36,740:INFO:Uploading model into container now
2021-02-16 10:31:36,741:INFO:create_model_container: 16
2021-02-16 10:31:36,741:INFO:master_model_container: 16
2021-02-16 10:31:36,741:INFO:display_container: 2
2021-02-16 10:31:36,741:INFO:<catboost.core.CatBoostRegressor object at 0x00000236EA431048>
2021-02-16 10:31:36,741:INFO:create_model() succesfully completed......................................
2021-02-16 10:31:37,789:INFO:SubProcess create_model() end ==================================
2021-02-16 10:31:37,789:INFO:Creating metrics dataframe
2021-02-16 10:31:37,877:INFO:Initializing create_model()
2021-02-16 10:31:37,878:INFO:create_model(estimator=<catboost.core.CatBoostRegressor object at 0x00000236EA431048>, fold=KFold(n_splits=10, random_state=786, shuffle=False), round=4, cross_validation=False, predict=False, fit_kwargs={}, groups=None, refit=True, verbose=False, system=False, metrics=None, display=None, kwargs={})
2021-02-16 10:31:37,878:INFO:Checking exceptions
2021-02-16 10:31:37,878:INFO:Importing libraries
2021-02-16 10:31:37,878:INFO:Copying training dataset
2021-02-16 10:31:37,879:INFO:Defining folds
2021-02-16 10:31:37,879:INFO:Declaring metric variables
2021-02-16 10:31:37,879:INFO:Importing untrained model
2021-02-16 10:31:37,879:INFO:Declaring custom model
2021-02-16 10:31:37,880:INFO:CatBoost Regressor Imported succesfully
2021-02-16 10:31:37,881:INFO:Cross validation set to False
2021-02-16 10:31:37,881:INFO:Fitting Model
2021-02-16 10:31:41,643:INFO:<catboost.core.CatBoostRegressor object at 0x00000236EA4FBA48>
2021-02-16 10:31:41,643:INFO:create_models() succesfully completed......................................
2021-02-16 10:31:42,685:INFO:create_model_container: 16
2021-02-16 10:31:42,685:INFO:master_model_container: 16
2021-02-16 10:31:42,685:INFO:display_container: 2
2021-02-16 10:31:42,685:INFO:<catboost.core.CatBoostRegressor object at 0x00000236EA4FBA48>
2021-02-16 10:31:42,685:INFO:compare_models() succesfully completed......................................
2021-02-16 10:31:42,698:INFO:Initializing create_model()
2021-02-16 10:31:42,698:INFO:create_model(estimator=rf, fold=None, round=4, cross_validation=True, predict=True, fit_kwargs=None, groups=None, refit=True, verbose=True, system=True, metrics=None, display=None, kwargs={})
2021-02-16 10:31:42,698:INFO:Checking exceptions
2021-02-16 10:31:42,699:INFO:Preparing display monitor
2021-02-16 10:31:42,739:INFO:Importing libraries
2021-02-16 10:31:42,740:INFO:Copying training dataset
2021-02-16 10:31:42,742:INFO:Defining folds
2021-02-16 10:31:42,743:INFO:Declaring metric variables
2021-02-16 10:31:42,761:INFO:Importing untrained model
2021-02-16 10:31:42,781:INFO:Random Forest Regressor Imported succesfully
2021-02-16 10:31:42,811:INFO:Starting cross validation
2021-02-16 10:31:42,812:INFO:Cross validating with KFold(n_splits=10, random_state=786, shuffle=False), n_jobs=-1
2021-02-16 10:31:55,776:INFO:Calculating mean and std
2021-02-16 10:31:55,780:INFO:Creating metrics dataframe
2021-02-16 10:31:55,812:INFO:Finalizing model
2021-02-16 10:31:57,613:INFO:Uploading results into container
2021-02-16 10:31:57,614:INFO:Uploading model into container now
2021-02-16 10:31:57,725:INFO:create_model_container: 17
2021-02-16 10:31:57,725:INFO:master_model_container: 17
2021-02-16 10:31:57,726:INFO:display_container: 3
2021-02-16 10:31:57,726:INFO:RandomForestRegressor(bootstrap=True, ccp_alpha=0.0, criterion='mse',
                      max_depth=None, max_features='auto', max_leaf_nodes=None,
                      max_samples=None, min_impurity_decrease=0.0,
                      min_impurity_split=None, min_samples_leaf=1,
                      min_samples_split=2, min_weight_fraction_leaf=0.0,
                      n_estimators=100, n_jobs=-1, oob_score=False,
                      random_state=786, verbose=0, warm_start=False)
2021-02-16 10:31:57,727:INFO:create_model() succesfully completed......................................
2021-02-16 10:31:58,703:INFO:Initializing plot_model()
2021-02-16 10:31:58,703:INFO:plot_model(plot=learning, fold=None, use_train_data=False, verbose=True, display=None, estimator=RandomForestRegressor(bootstrap=True, ccp_alpha=0.0, criterion='mse',
                      max_depth=None, max_features='auto', max_leaf_nodes=None,
                      max_samples=None, min_impurity_decrease=0.0,
                      min_impurity_split=None, min_samples_leaf=1,
                      min_samples_split=2, min_weight_fraction_leaf=0.0,
                      n_estimators=100, n_jobs=-1, oob_score=False,
                      random_state=786, verbose=0, warm_start=False), feature_name=None, fit_kwargs=None, groups=None, label=False, save=False, scale=1, system=True)
2021-02-16 10:31:58,704:INFO:Checking exceptions
2021-02-16 10:31:58,823:INFO:Preparing display monitor
2021-02-16 10:31:58,833:INFO:Preloading libraries
2021-02-16 10:31:58,947:INFO:Copying training dataset
2021-02-16 10:31:58,948:INFO:Copying test dataset
2021-02-16 10:31:58,949:INFO:Plot type: learning
2021-02-16 10:32:01,856:INFO:Fitting Model
2021-02-16 10:33:29,639:INFO:Visual Rendered Successfully
2021-02-16 10:33:30,611:INFO:plot_model() succesfully completed......................................
